site/zenodo_rdm/migrator/transform/entries/records/metadata.py

```diff
@@ -313,9 +313,7 @@ def transform(cls, entry):
 
         dates = cls._dates(entry.get("dates"))
         if dates:
-            # TODO uncomment after tested
-            # metadata.update({"dates": dates})
-            pass
+            metadata.update({"dates": dates})
```

Comment:
i added `return drop_nones(metadata)` at the end of the drafts entry. I wonder if it would be better to do the same for records. Would fix the custom fields empty keys issues and also save `if` clauses. WDYT?

Reply:
Yeah that's exactly the issue I was trying to bypass. So I agree with your suggestion